<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPoemInput</class>
 <widget class="QDialog" name="DialogPoemInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1582</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1561</width>
     <height>881</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainHLayout" stretch="24,1,24,1,24">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>*.PoemList</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="treePoems">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>*.UtilLog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableLog"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editNo">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editTitle">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editDate">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Rev</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editRev">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editContent">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editComment">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*.Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editDir">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="font">
           <font>
            <family>돋움</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QCheckBox" name="chkSaveAndClear">
          <property name="text">
           <string>Clear Content with saving data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="font">
         <font>
          <family>돋움</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>btn&amp;Save</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddPrevRev">
        <property name="text">
         <string>add previous rev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddNewRev">
        <property name="text">
         <string>add new rev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="font">
         <font>
          <family>돋움</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>clear content</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
